.github/workflows/overcloud-host-image-build.yml

```diff
@@ -143,7 +143,7 @@ jobs:
       - name: Install OpenStack client
         run: |
           source venvs/kayobe/bin/activate &&
-          pip install python-openstackclient
+          pip install python-openstackclient -c https://opendev.org/openstack/requirements/raw/branch/stable/${{ steps.openstack_release.outputs.openstack_release }}/upper-constraints.txt
```

Comment:
There is a shorter URL you can use: `http://releases.openstack.org/constraints/upper/${{ steps.openstack_release.outputs.openstack_release }}`